.github/fabricbot.json

```diff
@@ -8308,7 +5373,7 @@
               {
                 "name": "labelAdded",
                 "parameters": {
-                  "label": "needs more info"
```

Comment:
How did I miss that? 🙀  Thanks

Reply:
Yeah, I was surprised this "needs more info" had slipped through too. 🤷‍♂️ But we've fixed the glitch. :wink: